lib/perl/Genome/Qc/Tool.pm

```diff
@@ -50,4 +50,14 @@ sub output_file_accessor {
     return undef;
 }
 
+sub reference_build {
+    my $self = shift;
+    return $self->alignment_result->reference_build;
+}
+
+sub reference_sequence {
+    my $self = shift;
+    my $build =  $self->reference_build->full_consensus_path('fa');
```

Comment:
This might be clearer as something like:
```perl
my $build = $self->reference_build;
return $build->full_consensus_path('fa');
```
or just:
```perl
return $self->reference_build->full_consensus_path('fa');
```

The current version happens to return the FASTA.  It is a little misleading, though, since `$build` isn't a build and the variable isn't used.